tests/src/test/scala/matryoshka/instances/fixedpoint/partial.scala

```diff
@@ -69,7 +69,7 @@ class PartialSpec extends Specification with ScalazMatchers with ScalaCheck with
     "return after multiple runs" >> prop { (a: Conat, b: Conat) =>
       val (ai, bi) = (a.cata(height), b.cata(height))
       bi > 0 ==> {
-        val first = (a + b).transAna(Partial.delay(27)).runFor(ai)
+        val first = (a + b).transAna[Partial[Int]](Partial.delay(27)).runFor(ai)
```

Comment:
These operations (like the non-`trans` operations with the same names) now require a type annotation, because they only know that the result type has a particular functor, but not what proper type that functor has been extracted from.